lv_conf_template.h

```diff
@@ -133,8 +133,8 @@
     #endif
 #endif
 
-/* Use Arm-2D on Cortex-M based devices. Please only enable it for Helium Powered devices for now */
-#define LV_USE_DRAW_ARM2D 0
+/* Use Arm-2D to accelerate the sw render */
+#define LV_USE_DRAW_ARM2D_SYNC 0
```

Comment:
If it's used only with the SW render please move it to  e.g. line 103.

Reply:
OK, no problem